openpype/lib/local_settings.py

```diff
@@ -611,6 +611,12 @@ def get_openpype_username():
     settings and last option is to use `getpass.getuser()` which returns
     machine username.
     """
+
+    if AYON_SERVER_ENABLED:
+        import ayon_api
+
+        return ayon_api.get_user()["name"]
```

Comment:
Out of curiosity - what's the **name** of the user in Ayon API? Is it the actual full name like "Roy Nieterau" or a code equivalent "roy_nieterau" or is it a short version like "roy". how about special characters in certain names - is it a unicode equivalent or safe for filenames?

Reply:
It is username (regex: `^[a-zA-Z0-9][a-zA-Z0-9_\.\-]*[a-zA-Z0-9]$`. Full name is stored elsewhere.